.devcontainer/Dockerfile

```
@@ -0,0 +1,14 @@
+# Make sure RUBY_VERSION matches the Ruby version in .ruby-version
+ARG RUBY_VERSION=3.4.4
+FROM ghcr.io/rails/devcontainer/images/ruby:$RUBY_VERSION
+
+USER root
+ENV DEBIAN_FRONTEND=noninteractive
+
```

To make it run on my Windows machine, I need to set additional permissions.
This got it to work, but it doesn't seem like the best solution.

```suggestion

RUN mkdir -p /workspaces/openstreetmap-website \
    && chown -R vscode:vscode /workspaces/openstreetmap-website
```